Package.swift

```diff
@@ -5,29 +5,26 @@ import PackageDescription
 
 let package = Package(
     name: "MoEngage-iOS-SDK",
-    products: [
-        // Products define the executables and libraries a package produces, and make them visible to other packages.
-        .library(
-            name: "MoEngage-iOS-SDK",
-            targets: ["MoEngageCore","MoEngageSDK","MoEngageAnalytics","MoEngageMessaging","MoEngageObjCUtils", "MoEngageSecurity"])
-    ],
-    dependencies: [
-        // Dependencies declare other packages that this package depends on.
-        // .package(url: /* package url */, from: "1.0.0"),
-    ],
-    targets: [
-        .binaryTarget(name: "MoEngageSecurity", path: "./Frameworks/MoEngageSecurity.xcframework"),
-        
-        .binaryTarget(name: "MoEngageCore", path: "./Frameworks/MoEngageCore.xcframework"),
-        
-        .binaryTarget(name: "MoEngageAnalytics", path: "./Frameworks/MoEngageAnalytics.xcframework"),
-        
-        .binaryTarget(name: "MoEngageMessaging", path: "./Frameworks/MoEngageMessaging.xcframework"),
-        
-        .binaryTarget(name: "MoEngageObjCUtils", path: "./Frameworks/MoEngageObjCUtils.xcframework"),
+    platforms: [.iOS(.v11), .tvOS(.v11)],
+    products: [], dependencies: [], targets: [],
+    swiftLanguageVersions: [.v5]
+)
+
+let version = "9.16.2"
+let targetMap = [
+    "MoEngageAnalytics": "fd96db096d65f09f48b847278ecaa2e89bad68f201ea863a93de6c5cb5101ec1",
+    "MoEngageCore": "3d5db355583b59a1408fdc1671dd1eeef7ab2de36e7dafc1b53724b32495198a",
+    "MoEngageMessaging": "4facc247f91161d4511c8287a963603f578713d491aceeb24e5e4a5b7fc368ee",
+    "MoEngageObjCUtils": "372fa98ff57dbeb6c2ec8d843607a012d359c70dbeed99f51b6c9f64def4d62f",
+    "MoEngageSDK": "4ca5bed00858f4b69b72aa89cd11b82eed8d90903483f81f71966eea40cd4bf6",
+    "MoEngageSecurity": "a9fd0de77c6974491e4a9328d6dfa97bea90168a6cadc6e585747a29c72affcb",
+]
+
+for (target, checksum) in targetMap {
+    let url = "https://github.com/moengage/MoEngage-iOS-SDK/releases/download/\(version)/\(target).xcframework.zip"
+    package.targets.append(.binaryTarget(name: target, url: url, checksum: checksum))
+}
```

Comment:
its not possible to have multiple xcframeworks in one zip for SPM is it ?

Reply:
No SPM expects one xcframework per per zip

Reply:
If there are dependancy on a target, we should provide the framework zip path of that framework is it ?